TLM/TLM/State/OptionsTabs/GeneralTab_LocalisationGroup.cs

```diff
@@ -151,7 +151,12 @@ public static class GeneralTab_LocalisationGroup {
         }
 
         private static void OnRoadSignsThemeChanged(int newThemeIndex) {
-            if (!Options.IsGameLoaded()) {
+            if (!Options.IsGameLoaded(false)) {
+                // update global config only, skip reloading theme
+                var selectedTheme = RoadSignThemeManager.Instance.ThemeNames[newThemeIndex];
+                Main config = GlobalConfig.Instance.Main;
+                config.RoadSignTheme = selectedTheme;
+                GlobalConfig.WriteConfig();
                 return;
```

Comment:
this is duplicate code. instead you can guard the `switch` with `if`:
```cs
 if (Options.IsGameLoaded(false)) {
    // update global config only, skip reloading theme
    switch ( ... ) {
        ...
    }
}
```

Reply:
I'll leave that like it is and I see more bugs/issues/logic gaps with theme selection or forcing specific theme just by "use MPH" checkbox value above.